cocoa/src/quartzcore.rs

```diff
@@ -946,12 +946,12 @@ impl CALayer {
 
     #[inline]
     pub fn actions(&self) -> CFDictionary<CFStringRef, CFTypeRef> {
-        unsafe { msg_send![self.id(), actions] }
+        unsafe { CFDictionary::wrap_under_get_rule(msg_send![self.id(), actions]) }
```

Comment:
This is not returning a copy, so it is just a reference, but since we aren't returning a reference, we need to increment the refcount and manage it correctly.